install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -1,3 +1,30 @@
+{{- $defaultDisableCnpStatusUpdates := "false" -}}
+{{- $bpfMapDynamicSizeRatio := 0.0 -}}
+{{- $bpfMasquerade := "false" -}}
+{{- $bpfClockProbe := "false" -}}
+{{- $defaultIPAM := "hostscope" -}}
+{{- $defaultSessionAffinity := "false" -}}
+{{- $defaultOperatorApiServeAddr := "localhost:9234" -}}
+{{- $defaultBpfCtTcpMax := 524288 -}}
+{{- $defaultBpfCtAnyMax := 262144 -}}
+{{- if semverCompare ">=1.8" (default "1.8" .Values.upgradeCompatibility) -}}
```

Comment:
Nit: Some whitespace above might help to distinguish version-dependent sections for readability here.

Reply:
I wrongly assumed that it would lead to whitespace at the beginning of the file but that's not the case.